packages/packages/editor/src/components/__tests__/shell.test.tsx

```diff
@@ -0,0 +1,15 @@
+import { render } from '@testing-library/react';
+import Shell from '../shell';
+import { addToTop } from '../../locations';
+
+describe( '@elementor/editor Shell component', () => {
+	it( 'should render', () => {
+		addToTop( {
+			component: () => <div>test</div>,
+		} );
+
+		const { queryByText } = render( <Shell /> );
+
+		expect( queryByText( 'test' ) ).toBeTruthy();
```

Comment:
maybe test the `init()` method instead of testing the actual shell? (implementation details)

Reply:
Done

Reply:
Done